dev.py

```diff
@@ -432,6 +432,8 @@ def setup_build(args, env):
             return
     if args.werror:
         cmd += ["--werror"]
+    if args.gcov:
+        cmd += ['Db_coverage=true']
```

Comment:
@Smit-create it looks like you didn't actually test this, because it's missing a dash (it should be `-Db_coverage=true` to avoid seeing:
```
meson: error: unrecognized arguments: Db_coverage=true
Meson build setup failed! ({0} elapsed)
```

Reply:
Oops! sorry. I just missed that in some hurry.